### DISCUSSION TITLE
Why does \t in bound string display as 𝄚 ?

### DISCUSSION THREAD

**Discussion by moon6969** [AvaloniaUI/Avalonia#6421] [General]
Given View & ViewModel on Avolonia 0.10.6 :
```
<StackPanel Orientation="Vertical">
  <TextBox Text="{Binding backslash_T}" />
  <TextBox Text="{Binding actual_T}" />
  <TextBox Text="→\t← Text attribute with backslash t" />
  <TextBox Text="→ ← Text attribute with actual tab" />
</StackPanel>

public class MainWindowViewModel : ViewModelBase
{
    public string backslash_T => "→\t← Binding with backslash t";
    public string actual_T => "→  ← Binding with actual tab";
}
```
![AvaloniaTabTest3](https://user-images.githubusercontent.com/30491315/129456719-10842401-e3f5-48f5-b416-9d286e3545d5.png)

Why is the \t displaying as 𝄚 ? (I think it's unicode U+1D11A : MUSICAL SYMBOL FIVE-LINE STAFF)

Weirdly, if I ctrl+c copy the 𝄚 from the textbox it actually pastes as a tab (U+0009).

Not a big deal, but I prefer using \t in code so it's obvious that it's a tab rather than space.

**Comment by kekekeks** [comment]
XML doesn't follow C escape rules, use `&#x9;` instead.

**Reply by moon6969** [reply]
AH ok thanks.

**Reply by moon6969** [reply]
Although still doesn't explain why it displays as 𝄚 and copies as an actual Tab?

**Reply by moon6969** [reply]
`public string actual_T => "→&#x9;← Binding with actual tab";`
Just displays as
`→&#x9;← Binding with actual tab`
